Gemfile

```diff
@@ -2,22 +2,22 @@ source 'https://rubygems.org'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.1.10'
+gem 'rails', '~> 5'
```

Comment:
@case-eee This is the most dangerous thing I think I've ever done at a keyboard... Let me know if this was dumb enough that I should roll it back?